_data/sites.json

```diff
@@ -4306,6 +4306,18 @@
         ]
     },
 
+    {
+        "name": "Munzee",
+        "url": "https://www.munzee.com/privacy/",
+        "difficulty": "hard",
+        "notes": "Account deletion must be requested via email.",
+        "notes": "Send an e-mail to privacy@freezetag.com. This email should include your first name, last name, e-mail address and, if applicable, your social network ID for the social network from which you access these services. (for example, your Facebook user ID)",
```

Comment:
Notes are duplicated, please keep just one

Reply:
Didn't even notice that from the original PR... Good catch!